controllers/controller.js

```diff
@@ -53,4 +53,17 @@ exports.addComment = (request, response, next) => {
     })
     .catch(next)
 
-}
\ No newline at end of file
+}
+
+exports.getArticleToPatch = (request, response, next) => {
+    const {article_id} = request.params
+    checkArticleExists(article_id)
+    .then(() => {
+        return patchArticle(request.body, request.params)
+    })
+    .then((article) => {
+        response.status(200).send(article)
```

Comment:
This should be an object with a key of article and the article requested as a value